docs/development/buildenvironments.rst

```diff
@@ -16,20 +16,19 @@ any development on the Docker build system, you will need to set up `Docker`_ on
 your host system. Setup of Docker will vary by system, and so is out of the
 scope of this documentation.
 
-Once you have Docker set up, you will need to create the base image used for
-container creation. The ``base`` image is found in our `container images repo`_.
-It is the basic container image supported by our community site.
+Once you have Docker set up, you will need to pull down our build image. These
+images are found on our `Docker Hub repository`_, the source comes from our
+`container image repo`_.
 
-To get started, create the image using the `docker` command line tool. You can
-name the image whatever you like here, ``rtfd-build`` is the default name, but
-can be configured in your settings -- see `Configuration`_::
+To get started, pull one of the build images down::
 
-    docker build -t rtfd-build base/
+    docker pull readthedocs/build:latest
```

Comment:
Correct, this was me being lazy. Putting `2.0` there is likely to go stale, where `latest` wouldn't. Plus, I would like to point folks to test there, but that's perhaps a different issue.

I think you're right, I'm going to push up a fancier version.